### ISSUE TITLE
Inference results of the overfitted model are noise

### ISSUE THREAD

**Issue by kirilllzaitsev** [AssemblyAI-Examples/MinImagen#17]
Having an overfitted 'base' part of the Imagen, I try to sample with it for 300 timesteps and get the following:
![image](https://user-images.githubusercontent.com/44346535/233769943-2a357bbc-4584-4f6b-9a90-bbb163afb3d4.png)
During training, it predicts the noise well, and MSE goes down steadily. Somewhat similar is the following issue https://github.com/lucidrains/imagen-pytorch/issues/337. 

Could you give any advice on the direction to investigate the issue?

**Comment by chiho-5** [comment]
For how many epochs did you train. I think I'm encountering the same problem

**Comment by sadath-12** [comment]
same problem

**Comment by swayampragnya-malla** [comment]
same problem

**Comment by kirilllzaitsev** [comment]
> same problem

Try increasing the number of epochs (for instance, 1500 epochs (1 epoch = 1 gradient step) for a single image) and tuning the learning rate (starting with small values <1e-4)
